h2o-py/h2o/explanation/_explain.py

```diff
@@ -1480,27 +1589,33 @@ def _handle_orig_values(is_factor, tmp, encoded_col, plt, target, model, frame,
         # orig_null_value = tmp.from_num_to_factor(encoded_col)[tmp[encoded_col][idx]] if is_factor else tmp[encoded_col][idx]
         orig_null_value = PDP_RESULT_FACTOR_NAN_MARKER if is_factor else np.nan
         percentile_string = "for " + percentile_string if percentile_string is not None else ""
-        msg = "Original observation of \"{}\" {} is [{}, {}]. Plotting of NAs is not yet supported.".format(encoded_col, percentile_string, orig_null_value, tmp["mean_response"][idx])
+        msg = "Original observation of \"{}\" {} is [{}, {}]. Plotting of NAs is not yet supported.".format(encoded_col,
+                                                                                                            percentile_string,
+                                                                                                            orig_null_value,
+                                                                                                            tmp[
+                                                                                                                "mean_response"][
+                                                                                                                idx])
```

Comment:
I'd suggest keeping this in one line (`tmp["mean_response"][idx]`) even if it exceeds the line length limit.

Reply:
[a9dfa0d](https://github.com/h2oai/h2o-3/pull/6119/commits/a9dfa0dca78c37d998caa4dbd8d975cb361b9ea8)